Gemfile.lock

```diff
@@ -168,15 +167,15 @@ GEM
       http_parser.rb (>= 0.6.0)
     em-socksify (0.3.2)
       eventmachine (>= 1.0.0.beta.4)
-    erubis (2.7.0)
```

Comment:
:+1:

Just double checked this API wasn't being used. The only match was on the name `erubis` - but it's not impacted:

https://github.com/rapid7/metasploit-framework/blob/0d58014b7dcffe1ecd7c8b5ad6b87116af80ae08/modules/exploits/linux/http/github_enterprise_secret.rb#L116-L122